W 11398574 aal mfa (#218)
mulesoft/docs-access-management: modules/ROOT/pages/multi-factor-authentication.adoc

* Security key
* Salesforce Authenticator

== Multi-factor Authentication Requirements for Salesforce and MuleSoft Products

> * Beginning February 1, 2022, multi-factor authentication (MFA) must be enabled for all of your Salesforce users. To learn more about this requirement, see the https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce Multi-Factor Authentication FAQ].

Author: ```suggestion
* Beginning February 1, 2022, MFA must be enabled for all of your Salesforce users. To learn more about this requirement, see the https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce Multi-Factor Authentication FAQ].
```

> * Organizations created after April 30, 2022 require multi-factor authentication by default for all users.

Reviewer: multi-factor > MFA, since you already defined the acronym

Author: ```suggestion
* Organizations created after April 30, 2022 require MFA by default for all users.
```

> * Beginning July 21, 2022, user accounts that are added to an existing organization must configure MFA.

Author: ```suggestion
* Beginning July 23, 2022, user accounts that are added to an existing organization must configure MFA.
```
